app/main/helpers/dm_google_analytics.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+
+
+class CurrentProjectStageEnum(Enum):
+    SAVED = 'save_and_refine_search'
+    ENDED = 'search_ended'
+    DOWNLOADED = 'download_results'
+    READY_TO_ASSESS = 'ready_to_assess'
+    # the following correspond with digitalmarketplace-api:app.models.outcomes.Outcome.RESULT_CHOICES
+    FAILED = 'none-suitable'
+    CANCELLED = 'cancelled'
+    AWARDED = 'awarded'
```

Comment:
:crystal_ball: My crystal ball can see us needing to be able to refer to a generic project state from other apps, and when that happens we should probably move this and implement it in-api. Preferably as a `@hybrid_property`.

Also I'd personally call this idea "project current stage" because "current project stage" has the ambiguity over which of "(current project) stage" vs "current (project stage)" it means. Whereas "(project current)" clearly doesn't mean anything unless we're getting into the dried fruit business.

It's not super important though.

Reply:
I agree the project state should be in the api. This is a perfectly cromulent solution for now though.

Reply:
:jebediah_springfield: